src/Parser/Rogue/Subtlety/Modules/Legendaries/ShadowSatyrsWalk.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Wrapper from 'common/Wrapper';
+
+
+import EnergyTracker from '../RogueCore/EnergyTracker';
+
+class ShadowSatyrsWalk extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+		energyTracker: EnergyTracker,
+	};
+
+	on_initialized(){
+		this.active = this.combatants.selected.hasFeet(ITEMS.SHADOW_SATYRS_WALK.id);
+	}	
+
+	item() {
+		const builders = this.energyTracker.buildersObj;
+		if(!(SPELLS.SHADOW_SATYRS_WALK_ENERGY_BASE.id in builders)) return;
+
+		const base = builders[SPELLS.SHADOW_SATYRS_WALK_ENERGY_BASE.id];
+		const extra = builders[SPELLS.SHADOW_SATYRS_WALK_ENERGY_EXTRA.id];
+
+		const total = base.generated + extra.generated;
+		const avgExtra = extra.generated / base.casts;
+
+		
+    const totalPerMinute = (total / this.owner.fightDuration) * 1000 * 60;
+
+		return {
+			item: ITEMS.SHADOW_SATYRS_WALK,
+			result: <Wrapper>
+        <dfn data-tip={`Positioning at maximum range from your target could increase energy generation.`}>
```

Comment:
When there's no string merging it's preferred to use just "". e.g. `<dfn data-tip="Positioning at maximum range from your target could increase energy generation.">`